lumicks/pylake/population/mixture.py

```diff
@@ -62,22 +67,33 @@ def __init__(self, data, n_states, init_method="kmeans", n_init=1, tol=1e-3, max
             data = data.data
 
         self.n_states = n_states
-        self._model = GaussianMixture(
+        _model = GaussianMixture(
             n_components=n_states,
             init_params=init_method,
             n_init=n_init,
             tol=tol,
             max_iter=max_iter,
         )
         data = np.reshape(data, (-1, 1))
-        self._model.fit(data)
+        _model.fit(data)
+
+        # todo: remove when exit_flag is removed
+        self._deprecated_lower_bound = _model.lower_bound_
+
+        idx = np.argsort(_model.means_.squeeze())
+        self._model = ClassicGmm(
+            n_states,
+            _model.means_.squeeze()[idx],
+            1 / _model.covariances_.squeeze()[idx],
+            _model.weights_[idx],
+        )
 
         self._fit_info = PopulationFitInfo(
-            self._model.converged_,
-            self._model.n_iter_,
-            self._model.bic(data),
-            self._model.aic(data),
-            np.sum(self._model.score_samples(data)),
+            _model.converged_,
+            _model.n_iter_,
+            _model.bic(data),
+            _model.aic(data),
+            np.sum(_model.score_samples(data)),
```

Comment:
Very minor, but might be nice to assign these with keyword args, since they're spread out over lines anyhow. Or maybe just the last one at least. Then you immediately see that it is the log likelihood.